src/clients/SnapshotGraphql.ts

```diff
@@ -374,4 +394,48 @@ export class SnapshotGraphql extends API {
   async getPendingProposals(start: Date, end: Date, fields: (keyof SnapshotProposal)[], limit = 1000) {
     return await this.fetchProposals({ start, end, fields, scoresState: SnapshotScoresState.Pending }, 0, limit)
   }
+
+  async getVpDistribution(address: string): Promise<VpDistribution> {
+    const query = `
+     query getVpDistribution($space: String!, $voter: String!){
+        vp (
+          voter: $voter,
+          space: $space
+        ) {
+          vp
+          vp_by_strategy
+        } 
+      }
+    `
+    const variables = {
+      space: SNAPSHOT_SPACE,
+      voter: address,
+    }
+
+    const result = await this.fetch<SnapshotVpResponse>(
+      GRAPHQL_ENDPOINT,
+      this.options().method('POST').json({
+        query,
+        variables: variables,
+      })
+    )
+
+    if (!result?.data?.vp) {
+      throw Error('Unable to fetch VP Distribution')
+    }
+
+    const vpByStrategy = result?.data?.vp.vp_by_strategy
+
+    return {
+      totalVp: Math.floor(result?.data?.vp.vp),
+      ownVp: Math.floor(result?.data?.vp.vp - vpByStrategy[5]),
+      wManaVp: Math.floor(vpByStrategy[0]),
```

Comment:
We can have an index mapper... something like... IDK if we are going to use or we are using this strategy order somewhere else already.

```
enum StrategyOrder {
 WrappedMana,
 Land,
 Estate,
 ...
}

... 
vpByStrategy[StrategyOrder.WrappedMana]
```

Reply:
Love this. Let's make it explicit